benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -357,7 +357,7 @@ msgstr "Provide your bank card details"
 
 #: benefits/eligibility/views.py:75
 msgid "eligibility.pages.start.bankcard.text"
-msgstr "Must be a contactless debit or credit card issued by Visa or Mastercard."
+msgstr "Must be a contactless debit or credit card by Visa or Mastercard."
```

Comment:
Was the corresponding Spanish text adjusted?

Reply:
The Spanish text never had a word for `issued`. We took out `issued` in English purely for spacing reasons. 

<img width="744" alt="image" src="https://user-images.githubusercontent.com/3673236/169860667-aec27582-e9a8-4884-b3c0-46b69ff537d3.png">

[msgstr "Debe ser una tarjeta de débito o crédito sin contacto de Visa o Mastercard."](https://github.com/cal-itp/benefits/blob/45ea67396e4679c1b108fcc06a9e314073c10b7e/benefits/locale/es/LC_MESSAGES/django.po#L345)